src/Apps/Artwork/Components/__tests__/ArtworkBanner.test.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  ArtworkAuctionBannerFixture,
+  ArtworkFairBannerFixture,
+  ArtworkUpcomingShowBannerFixture,
+  ArtwrorkNoBannerFixture,
+} from "Apps/__test__/Fixtures/Artwork/ArtworkBanner"
+import { ArtworkBannerFragmentContainer } from "Apps/Artwork/Components/ArtworkBanner"
+import { renderRelayTree } from "DevTools"
+import { GraphQLResolveInfo } from "graphql"
+// import { cloneDeep } from "lodash"
+import { graphql } from "react-relay"
+
+jest.unmock("react-relay")
+
+describe("ArtworkBanner", () => {
+  const getWrapper = async (response = ArtwrorkNoBannerFixture) => {
+    return await renderRelayTree({
+      Component: ArtworkBannerFragmentContainer,
+      query: graphql`
+        query ArtworkBanner_Test_Query {
+          artwork(id: "richard-anuszkiewicz-lino-yellow-318") {
+            ...ArtworkBanner_artwork
+          }
+        }
+      `,
+      mockResolvers: {
+        Artwork: () => ({
+          ...response,
+          // FIXME: Need to figure out how to return proper values for aliesed objects
+          context: (_source, _args, _context, info: GraphQLResolveInfo) => {
+            const alias = info.fieldNodes[0].alias!.value
+            const x = response[alias]
+            return x
+          },
+        }),
+      },
+    })
+  }
+  let wrapper
+
+  describe("ArtworkBanner for artwork with no banner", () => {
+    beforeAll(async () => {
+      wrapper = await getWrapper()
+    })
+
+    it("renders nothing", () => {
+      const html = wrapper.html()
+      expect(html).toBe(null)
+    })
+  })
+
+  describe("ArtworkBanner for artwork with auction banner", () => {
+    beforeAll(async () => {
+      wrapper = await getWrapper(ArtworkAuctionBannerFixture)
+    })
+
+    it("renders a correct data for the auction", () => {
+      const html = wrapper.html()
+      expect(html).toContain("In auction")
+      // expect(html).toContain("Doyle: Post-War & Contemporary Art")
```

Comment:
@alloy - even though the alias seemed to be picked up properly now, the final data does not include object that is inside of the context data. I suspect it is because context itself in turn loads either Auction/Fair or Show and appends data there. Not sure how to fix that part.